src/Whoops/Exception/Frame.php

```diff
@@ -58,7 +58,9 @@ public function getFile($shortened = false)
         if ($shortened && is_string($file)) {
             // Replace the part of the path that all frames have in common, and add 'soft hyphens' for smoother line-breaks.
             $dirname = dirname(dirname(dirname(dirname(dirname(dirname(__DIR__))))));
-            $file = str_replace($dirname, "&hellip;", $file);
+            if ($dirname !== '/') {
+                $file = str_replace($dirname, "&hellip;", $file);
```

Comment:
Now that `TemplateHelper::shorten` is added, this here should not have repeated code, but should use that instead.

- [ ] Replace this repeated part with a call to `TemplateHelper::shorten`